ducktape/src/main/scala/io/github/arainko/ducktape/internal/macros/ProductTransformations.scala

```diff
@@ -132,22 +132,21 @@ private[ducktape] object ProductTransformations {
   private def fieldTransformations[Source: Type](
     sourceValue: Expr[Source],
     fieldsToTransformInto: List[Field]
-  )(using Quotes, Fields.Source) = {
+  )(using Quotes, Fields.Source): List[quotes.reflect.NamedArg] = {
     import quotes.reflect.*
 
     fieldsToTransformInto.map { field =>
-      field ->
-        Fields.source
-          .get(field.name)
-          .getOrElse(Failure.abort(Failure.NoFieldMapping(field.name, Type.of[Source])))
+      field -> Fields.source
+        .get(field.name)
+        .getOrElse(Failure.abort(Failure.NoFieldMapping(field.name, Type.of[Source])))
```

Comment:
```suggestion
        .getOrElse(Failure.abort(Failure.NoFieldMapping(field.name, summon[Type[Source]])))
```
okay now, this one's on me but since it's part of the diff... haha